Podfile

```diff
@@ -40,4 +41,14 @@ end
 
 post_install do |installer|
   system("sh tools/generateAcknowledgements.sh")
+
+  # convert incompatible pods back to Swift 3.2
```

Comment:
Worth a local fork of this to fix this issue? I'm saying this as it might also be worth updating the local highlight.js dependency (https://github.com/raspu/Highlightr/tree/master/Pod/Assets/Highlighter) to ensure we have the latest styles and syntax!

(Project hasn't been updated in like a year!)

Reply:
The version in Highlighter is 9.9.0, in the latest (9.12.0) has fixes to Swift

https://highlightjs.org/

So yeah should probs update! 😄

Reply:
Good call. Let's do that in a followup, want to collect some feedback from beta testers and make sure this thing is stable. Going to merge as-is, but then we can create a local pod and manually fix as we see fit.